sql/size_utils.sql

```diff
@@ -92,7 +92,9 @@ $BODY$
             0::BIGINT AS compressed_toast_size,
             0::BIGINT AS compressed_heap_size
         FROM
-            _timescaledb_catalog.hypertable
+            _timescaledb_catalog.hypertable ht
+            INNER JOIN pg_class c ON relname = ht.table_name
```

Comment:
```suggestion
            INNER JOIN pg_class c ON relname = ht.table_name AND c.relkind = 'r'
```
To read only tables from catalog.

Reply:
Done.